public/locales/es/validate.json

```diff
@@ -122,11 +122,11 @@
   },
   "mission-complete": {
     "title": "¡Gran trabajo!",
-    "body-1": "¡Acabas de validar {{n}} etiquetas de Rampas Peatonales!",
-    "body-2": "Acabas de validar {{n}} etiquetas de Rampas Peatonales Ausentes!",
-    "body-3": "Acabas de validar {{n}} etiquetas de Obstáculos!",
-    "body-4": "Acabas de validar {{n}} etiquetas de Problemas en Superficie!",
-    "body-7": "Acabas de validar {{n}} etiquetas de que No Hay Banqueta!",
+    "body-1": "¡Acabas de validar {{n}} etiquetas de Rampas Peatonales.",
```

Comment:
Missing the "¡" here. There was a translation error where we only had that for this line before, whoops!